specification/configuration.md

```diff
@@ -163,6 +163,27 @@ Other requirements:
 - RUM library MUST use the Zipkin v2 JSON span exporter by default
 - RUM library MUST limit the number of sent spans to 100 in a 30 second window per `component` attribute value
 
+#### Serverless 
+
+Current serverless offering is composed of separate metrics (non OpenTelemetry) and traces (OpenTelemetry) solutions. By default, the serverless components send data directly to Splunk Observability Cloud (direct ingest). 
+   
+Apart from standard set of configuration properties for instrumentation libraries based on OpenTelemetry, serverless MUST honour the following:  
```

Comment:
In addition to honoring, I think we should require token+realm or endpoint to be defined.

Reply:
Added remark.